src/.pylintrc

```diff
@@ -444,8 +444,7 @@ timeout-methods=requests.api.delete,requests.api.get,requests.api.head,requests.
 
 # List of note tags to take in consideration, separated by a comma.
 notes=FIXME,
-      XXX,
-      TODO
+      XXX
```

Comment:
Are we deciding to remove TODO from this list? I think I remember you being the one to add it, so I'm interested to hear your thoughts

Reply:
The list is the set of terms that we DO throw pylint warnings for. By removing it from this list, they're now allowed in comments without pylint complaining.

https://github.com/lincc-frameworks/python-project-template/pull/343/files